src/cmd-osbuild

```diff
@@ -405,9 +405,23 @@ main() {
     else
         cmd="runvm_with_cache"
     fi
-    $cmd -- /usr/lib/coreos-assembler/runvm-osbuild                                                     \
+
+    # Use the bootc install to-filesystem manifest for rawhide
+    # See discussion in https://github.com/coreos/coreos-assembler/pull/4224#pullrequestreview-3076134370
+    # Remove when we switch to use bootc install to-filesystem to all streams
+    releasever=${build%%.*}
+    with_bootc_install=""
+    if [ "$releasever" -ge 44 ]; then
+        with_bootc_install=".bootc"
+    fi
```

Comment:
This is effective, but brittle. It would probably give us the most flexibility if we made it configurable via an environment variable and also possibly a manifest setting like was done over in
https://github.com/coreos/fedora-coreos-config/pull/3754 and https://github.com/coreos/coreos-assembler/commit/532225d620e57b90d7b804d57177fa4af15bb925

i.e. I could see us wanting to roll this out in `next` and also RHCOS 10/SCOS 10 first and having those knobs would help that.